dockerfiles/ubuntu/is-analytics/init.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/sh
+# ------------------------------------------------------------------------
+# Copyright 2018 WSO2, Inc. (http://wso2.com)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+# ------------------------------------------------------------------------
+set -e
+
+# product profile variable
+wso2_server_profile=is-analytics
+
+# custom WSO2 non-root user and group variables
+user=wso2carbon
+group=wso2
```

Comment:
We usually use same value for user and its default group. Shall we use 'wso2' for both?

Reply:
Hi Kasunbg, 
"wso2carbon" is the standard username that we have been using for a while and "wso2" is the group name.